xen/arch/arm/coproc/coproc.c

```diff
@@ -21,6 +21,7 @@
 #include <xen/err.h>
 #include <xen/guest_access.h>
 #include <xen/keyhandler.h>
+#include <xen/vmap.h>
 
 #include "coproc.h"
 
```

Comment:
I know that we have 4 platform ops at the moment and all of them seem to be mandatory. But, I think that you should add check for particularly operation to be present before call it.
e.x. 
if (coproc->ops->ctx_switch_from)
   coproc->ops->ctx_switch_from(curr);